app/src/components/account/Paywall.vue

```diff
@@ -0,0 +1,320 @@
+<template>
+  <div class="paywall-container">
+    <h3>
+      Lunie Notifications<br />
+      are here!
+    </h3>
+    <TmBtn
+      id="getStartedBtn"
+      value="Get Started"
```

Comment:
@jbibla This should say something about Premium

Reply:
i think it's fine - after you click it you'll see this modal: 

<img width="683" alt="Screen Shot 2020-07-06 at 8 01 12 PM" src="https://user-images.githubusercontent.com/6021933/86671238-732bce00-bfc3-11ea-9e3b-413c7fa883da.png">